src/pages/updates/work-in-progress/index.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Work in progress
+description:
+  This page provides early visibility for new Carbon concepts and current work
+  in progress.
+---
+
+<PageDescription>
+
+An opportunity to see early Carbon concepts and ideas, and experiment with work
+in progress.
+
+</PageDescription>
+
+This work in progress is not supported, and is not approved for production use.
+The Carbon team gratefully accepts feedback or comment.
+
+| Tools and resources               |                                                    |
+| --------------------------------- | -------------------------------------------------- |
+| [Carbon Sketch kit for mobile](/) | Intro sentence about the Carbon mobile sketch kit. |
+| [Data visualization guidance](/)  | Intro about data viz file.                         |
+
+| Components and patterns                                                                                                |                                                                                                                 |
+| ---------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
+| [Tree view component](https://deploy-preview-6008--carbon-components-react.netlify.app/?path=/story/treeview--default) | Intro sentence about the tree view. [Github issue](https://github.com/carbon-design-system/carbon/issues/5180). |
+| [Text toolbar](/)                                                                                                      | Intro sentence about the text toolbar.                                                                          |
+
+| Carbon website concepts       |                                       |
+| ----------------------------- | ------------------------------------- |
+| [Information architecture](/) | Info about IA exploration.            |
+| [Home page concepts](/)       | Info about home page concepts.        |
+| [Community index](/)          | Info about community component index. |
```

Comment:
```suggestion
| [Community index](/)          | The community component index surfaces components that IBM teams have built, so that others can see and reuse them. |
```